storage/badger/blocks.go

```diff
@@ -108,6 +108,17 @@ func (b *Blocks) IndexBlockForCollections(blockID flow.Identifier, collIDs []flo
 	return nil
 }
 
+// InsertLastFullBlockHeight inserts the last full block height
+func (b *Blocks) InsertLastFullBlockHeight(height uint64) error {
+	return operation.RetryOnConflict(b.db.Update, func(tx *badger.Txn) error {
+		err := operation.SetLastCompleteBlockHeight(height)(tx)
```

Comment:
Should we use `InsertLastCompleteBlockHeight` instead?

Reply:
No, I renamed SetLastCompletedBlockHeight -> InsertLastCompletedBlockHeightIfNotExists https://github.com/onflow/flow-go/pull/2974/commits/5afa8ef0dc376fd24ffc268351fc7b5b28676242